### DISCUSSION TITLE
How do you add a new language in your own project?

### DISCUSSION THREAD

**Discussion by Matthias-DS** [rsuite/rsuite#3023] [Q&A]
I copied the js file from /node_modules/rsuite/esm/locales/<language>.js and translated it to fit my needs. That works excellently except I cannot set the months. How do you translate the months in a dateRangePicker component?

**Comment by simonguo** [comment]
Hi @cabaj1  You can refer to the language files in this directory. 
https://github.com/rsuite/rsuite/tree/main/src/locales

- [Guidance for new languages added to i18n](https://rsuitejs.com/zh/guide/i18n/).
- [How to add new languages to rsuite? ](https://github.com/rsuite/rsuite/discussions/2927)
